tests/public_data/test_publish_public_data_json_script.py

```diff
@@ -46,12 +46,6 @@ def test_script_incremental_query(
 
         assert res.returncode == 0
 
-        with pytest.raises(NotFound):
-            temp_table = (
-                f"{project_id}.{temporary_dataset}.incremental_query_v1_20200315_temp"
-            )
-            bigquery_client.get_table(temp_table)
-
```

Comment:
This was checking if the temporary table gets removed. Since the table name now has some random string in it, it's the actual table name is unknown and cannot be checked here :|